examples/sveltekit-pwa/pwa-configuration.js

```diff
@@ -62,16 +62,22 @@ const workboxOrInjectManifestEntry = {
 		// manifest.webmanifest is added always by pwa plugin, so we remove it.
 		// EXCLUDE from the sw precache sw and workbox-*
 		const manifest = entries.filter(({ url }) =>
-			url !== 'manifest.webmanifest' && !url.endsWith('sw.js') && !url.startsWith('workbox-')
+			!url.endsWith('sw.js') && !url.startsWith('workbox-')
 		).map((e) => {
 			let url = e.url
-			if (url && url.endsWith('.html')) {
+			if (!url)
+				return e
+
+			if (url.endsWith('.html')) {
 				if (url.startsWith('/'))
 					url = url.slice(1)
 
 				e.url = url === 'index.html' ? '/' : `/${url.substring(0, url.lastIndexOf('.'))}`
 				console.log(`${url} => ${e.url}`)
 			}
+			else if (url === '_app/immutable/manifest.webmanifest') {
```

Comment:
the `immutable` thing should be fixed now?

Reply:
thx @benmccann I'll change this tmr with the changes added to `pwa-sveltekit-vite` repo changes, awaiting for the sequential plugin behavior...

Reply:
I'll add also workbox options to change some prefixes automatically: https://github.com/userquin/pwa-sveltekit-vite/blob/master/vite.config.js#L50

We can also add multiple globaPatterns, and so, we directly exclude the server folder in the output.